ext/TransitionVisualizations.jl

```diff
@@ -0,0 +1,229 @@
+module TransitionVisualizations
+
+using TransitionsInTimeseries, Makie
+
+# Default options for plotting utilities
+const default_accent_linewidth = 3
+const default_colors = ["#7143E0", "#0A9A84", "#191E44", "#AF9327", "#701B80", "#2E6137"]
+
+default_indicator_label(res::IndicatorsChangesResults) = [string(nameof(
+    res.config.indicators[i])) for i in eachindex(res.config.indicators)]
+
+function default_chametric_label(res::IndicatorsChangesResults)
+    labels = String[]
+    for i in eachindex(res.config.change_metrics)
+        if res.config.change_metrics[i] isa PrecomputedRidgeRegressionSlope ||
+            res.config.change_metrics[i] isa RidgeRegressionSlope
+            push!(labels, "Regression slope")
+        else
+            push!(labels, string(nameof(res.config.change_metrics[i])))
+        end
```

Comment:
This is not maintainable code. We can't afford to add an if clause for any particular change metric. Why is this important here, and why is not important for e.g. difference of means (which in my line of work is a much more freqquently used metric)? 

We should have the same code everwhere with string name of. A user should be able to override a keyword andp provide a vector of strings as names.

Reply:
I agree with you that this is not well-maintainable code. Shorting the names of the regression indicators is however important because simply calling `nameof(cha_metric)` gives a very long string. This is now much improved by replacing the bit you highlighted by:

```
default_chametric_label(res::IndicatorsChangesResults) = [shortname(
    cha_metric) for cha_metric in res.config.change_metrics]

shortname(metric) = string(nameof(metric))
shortname(metric::PrecomputedRidgeRegressionSlope) = "Regression slope"
shortname(metric::RidgeRegressionSlope) = "Regression slope"
```